setup.cfg

```diff
@@ -1,10 +1,11 @@
-[isort]
-combine_as_imports = true
-default_section = THIRDPARTY
-include_trailing_comma = true
-line_length = 79
-multi_line_output = 5
-not_skip = __init__.py
-known_standard_library = marshal
-known_first_party = voc
-sections=FUTURE,STDLIB,THIRDPARTY,FIRSTPARTY,LOCALFOLDER
+[flake8]
+# https://flake8.readthedocs.org/en/latest/
+exclude=settings.py,wsgi.py,manage.py,*/migrations/*
+max-complexity = 10
+max-line-length = 119
```

Comment:
Habit, mostly. IIRC, editors in the Good Ole' Days used to show a continuation mark in the 80th column, so you set the line length at 79, and if you ever saw that mark, you knew you had a problem.